tracer/test/Datadog.Trace.Tests/Telemetry/config_prefix_block_list.json

```diff
@@ -122,6 +122,8 @@
   "DD_AwsKinesis_",
   "DD_TRACE_AzureServiceBus_",
   "DD_AzureServiceBus_",
+  "DD_TRACE_Random_",
+  "DD_Random_",
```

Comment:
Don't forget to update the master list in dd-go to 🙂

Reply:
oh, and if you change the integrationID, you'll need to change this list! :)

Reply:
Right! Thanks!